src/web-ui/src/repositories/locationsRepository.js

```diff
@@ -16,5 +16,9 @@ const resource = "/locations";
 export default {
     get() {
         return connection.get(`${resource}/all`)
+    },
+
+    getLocationsByBeer(beerName) {
+        return connection.get(`${resource}?beerName=${beerName}`)
```

Comment:
I was thinking we could use a path parameter instead of a query one. This would follow what has been done for other services (e.g. Products service endpoint for getting all products within a category is `/products/category/{categoryName}`)

In this case, something like `/locations/beer/{beerName}`

Reply:
Oh weird. This is very inconsistent across services then 🙂